### PR TITLE
Add website relation back in

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm-merge/19/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm/23/
Test FAILed.

File: reactive/bundleservice_charm.py [hunk 1]
```diff
@@ -26,6 +25,11 @@ def setup_nagios(nagios):
     set_state('bundleservice-charm.nrpe-check-added')
 
 
+@when('website.available')
+def configure_website(website):
+    website.configure(port=hookenv.config('listen-port'))
+
```
Comment (jcsackett) [inline reactive/bundleservice_charm.py:33]: You might consider having this set a state like `website.configured` and then adding a `when_not` to this function on that state, and then creating the logic to unset the configured state when a relation parts.

On the other hand, the cost in this instance of redoing the configuration every time the hook fires is low, so you certainly don't _need_ to.

**Comment by jcsackett** [comment]
LGTM, as the CI failure is expected.

You can implement my note or not as you see fit. As I said, it's hardly necessary, but something to think about.

:+1:

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm-merge/20/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm/25/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm-merge/21/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm/26/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm/27/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-charm/28/
Test PASSed.

**Comment by hatched** [comment]
:+1: Thanks @makyo looks good!

**Comment by makyo** [comment]
Big green button until we get the merge job working

**Pull request merged**